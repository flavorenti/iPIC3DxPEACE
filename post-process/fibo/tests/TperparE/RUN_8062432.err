slurmstepd-irene8001: error: task/cgroup: task[0] unable to set taskset '0x0'
+ SCRIPT_PID=71289
+ /bin/bash -x /tmp/jobstart.71245
+ set +x
+ unset _mlshdbg
+ '[' 1 = 1 ']'
+ case "$-" in
+ set +x
+ unset _mlshdbg
+ ml python
+ module ml python
+ unset _mlshdbg
+ '[' 1 = 1 ']'
+ case "$-" in
+ set +x
load module python/2.7.14 (Python)
+ unset _mlshdbg
+ return 0
+ set -x
+ ccc_mprun -E--exclusive -n 1 python -u test_TperparE.py /ccc/scratch/cont005/gen12428/lavorenf/Mercury_SaeInit/PR0/Normal/run2/old_bc/data
+ wait
+ ccc_mprun -E--exclusive -n 1 python -u test_TperparE.py /ccc/scratch/cont005/gen12428/lavorenf/Mercury_SaeInit/PR0/minusBz/run1_oldbc/data
Traceback (most recent call last):
  File "test_TperparE.py", line 54, in <module>
    from_VTK.get_meta(silent=False)
  File "/ccc/cont005/home/oca/lavorenf/ipic3d-klm_Irene/post-process/fibo/mod_from.py", line 50, in get_meta
    with open(os.path.join(self.address,extra_address,'SimulationData.txt'),'r') as foo:
IOError: [Errno 2] No such file or directory: '/ccc/scratch/cont005/gen12428/lavorenf/Mercury_SaeInit/PR0/Normal/run2/old_bc/data/SimulationData.txt'
srun: error: irene8001: task 0: Exited with exit code 1
+ exit 0
