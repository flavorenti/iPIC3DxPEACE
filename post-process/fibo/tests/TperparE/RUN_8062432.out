('iPIC3D> cell number               :  ', (256, 208, 208))
('iPIC3D> domain size               :  ', (25.0, 20.0, 20.0))
('iPIC3D> mesh spacing              :  ', (0.09765625, 0.09615384615384616, 0.09615384615384616))
('iPIC3D> periodicity               :  ', (False, False, False))
('iPIC3D> time step                 :  ', 0.5)
('iPIC3D> species                   :  ', ['e0', 'i1'])
('          e0 charge-over-mass           :  ', -100.0)
('          i1 charge-over-mass           :  ', 1.0)
('LOAD->', 5000)
['Je_y00005000', 'Je_z00005000', 'Je_x00005000']
('COMPUTE->', 5000)
['Je_y00005000', 'Jegf_y00005000', 'Je_x00005000', 'Jegf_z00005000', 'Je_z00005000', 'Jegf_x00005000']
('PRINT->', 5000)
('DELETE->', 5000)
COMP.TIME [seg5000] = 59.174 s

########## ########## ########## ########## ########## ########## ########## ##########
Execution Sum Up
########## ########## ########## ########## ########## ########## ########## ##########
Jobid     : 8062432
Jobname   : TperparE
User      : lavorenf
Account   : gen12428@xlarge
Limits    : time = 2:47:00 , memory/task = 26800 Mo
Date      : submit = 20/01/2022 11:32:45 , start = 20/01/2022 11:32:45 , end = 20/01/2022 14:19:45
Execution : partition = xlarge , QoS = normal@xlarge , Comment = (null)
Resources : ntasks = 2 , cpus/task = 6 , ncpus = 12 , nodes = 1
   Nodes=irene8001 CPU_IDs=44-55 Mem=321600 GRES=
 
Memory / step 
--------------
                          Resident Size (Mo)                     Virtual Size (Go)
JobID            Max     (Node:Task)       AveTask    Max  (Node:Task)            AveTask
-----------      ------------------------  -------    --------------------------  -------

Accounting / step 
------------------

JobID           JobName                  Ntasks  Ncpus Nnodes  Layout       Elapsed               User         System     Eff  State
------------    ------------             ------  ----- ------  -------      -------               -------      ------     ---  -----
8062432         TperparE                      -     12      1        -     00:08:17   (100%)            -            -      -  -
8062432.batch   batch                         1     12      1  Unknown     00:08:17   (100%)     00:00:00     00:00:00      0  RUNNING
8062432.extern  extern                        1     12      1  Unknown     00:08:17 (100.0%)     00:00:00     00:00:00      0  RUNNING
8062432.0       python                        1      6      1   BBlock     00:08:09  (98.3%)     00:00:05     00:00:02     .2  COMPLETED
8062432.1       python                        1      6      1   BBlock     00:07:32  (90.9%)     00:00:01     00:00:01      0  FAILED
########## ########## ########## ########## ########## ########## ########## ##########
